app/api/schema/users.py

```diff
@@ -49,6 +49,12 @@ class Meta:
     password = fields.Str(required=True, load_only=True)
     is_super_admin = fields.Boolean(dump_only=True)
     is_admin = fields.Boolean(dump_only=True)
+    is_user_organizer = fields.Boolean(dump_only=True)
```

Comment:
isn't `_user` here redundant? e.g. `is_organizer` would be sufficient, right?

Reply:
`is_organizer` is already a method in user model, whose purpose is to check whether user is a organizer of a event (event_id required) but here `is_user_organizer` we want to just check whether a user is organizer of any of the event in history.